docker/m3dbnode/development.Dockerfile

```diff
@@ -1,4 +1,8 @@
+<<<<<<< Updated upstream
 FROM alpine:latest
+=======
+FROM alpine:3.11
+>>>>>>> Stashed changes
```

Comment:
Should probably clean up these git artefacts

Reply:
🤦‍♂

Reply:
Yup - sg. Let's use alpine 3.11?

Perhaps we should upgrade to 3.11 everywhere too. i.e. find a replace all of `alpine3.9` with `alpine3.11` and also update all `alpine:latest` with `alpine:3.11` so we have a single version everywhere.